app/views/projects/automated_workflow_executions/_edit_dialog.html.erb

```diff
@@ -0,0 +1,14 @@
+<%= viral_dialog(open: true, size: :extra_large) do |dialog| %>>
+  <%= dialog.with_header(title: t(".title", workflow_name: @automated_workflow_execution.name)) %>
```

Comment:
**nitpick (blocking):** If `name` isn't set can you pass `id` instead.

Reply:
Updated in d4fb7f7